testing/smoke-test/build.gradle.kts

```diff
@@ -59,6 +61,18 @@ dependencies {
     smokeTestDistributionRuntimeOnly(projects.distributionsFull)
 }
 
+androidHomeWarmup {
+    sdkVersions.set(
```

Comment:
💅 Maybe `./gradle/dependency-management/agp-versions.properties` could be sourced here, but it doesn't contain all the information needed